src/FEniCS.jl

```diff
@@ -1,15 +1,17 @@
-#__precompile__() #precompilation is not currently working.Possible pointer error?
+ #__precompile__() #precompilation is not currently working.Possible pointer error?
```

Comment:
It's because the `@pyimport` stores a pointer. Precompiling pointers will fail. Use an init function for it to precompile:

https://github.com/JuliaDiffEq/PyDSTool.jl/blob/master/src/PyDSTool.jl#L9